crates/api_models/src/payments.rs

```diff
@@ -1410,12 +1410,14 @@ pub struct OrderDetails {
     /// The quantity of the product to be purchased
     #[schema(example = 1)]
     pub quantity: u16,
+    /// the amount per quantity of product
+    pub amount: i64,
 }
 
 #[derive(Default, Debug, Eq, PartialEq, serde::Deserialize, serde::Serialize, Clone, ToSchema)]
 pub struct Metadata {
     /// Information about the product and quantity for specific connectors. (e.g. Klarna)
-    pub order_details: Option<OrderDetails>,
+    pub order_details: Option<Vec<OrderDetails>>,
```

Comment:
is this tested for both klarna and zen?

Reply:
This is tested for klarna and adyen, @rishavkar can confirm if this is tested for zen connector

Reply:
What happens to existing orders in sandbox? it will still try to fetch as single OrderDetails right?
don't have to write migration to fix that?

Reply:
- i have tested for zen and adyen
- sandbox has entries with order_details as objects....so migration would be needed ideally.... would i have to add the migration script in this PR? 
@jagan-jaya

Reply:
Yes, Add the migration and steps needs to done before and after the deployment.

Is the changes are back-compatible?